examples/Bremsstrahlung/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2013-2017 Rene Widera, Benjamin Worpitz, Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+#include "simulation_defines.hpp"
+#include "particles/Identifier.hpp"
+#include "compileTime/conversion/MakeSeq.hpp"
+#include "identifier/value_identifier.hpp"
+
+#include "particles/traits/FilterByFlag.hpp"
+#include "particles/Particles.hpp"
+#include <boost/mpl/string.hpp>
+
+#include "particles/ionization/byField/ionizers.def"
+
+namespace picongpu
+{
+
+/*########################### define particle attributes #####################*/
+
+/** describe attributes of a particle*/
+using DefaultParticleAttributes = MakeSeq_t<
+    position< position_pic >,
+    momentum,
+    weighting,
+    particleId,
+    momentumPrev1
+#if( RAD_MARK_PARTICLE > 1 ) || ( RAD_ACTIVATE_GAMMA_FILTER != 0 )
+    , radiationFlag
+#endif
+>;
+
+/*########################### end particle attributes ########################*/
+
+/*########################### define species #################################*/
+
+/*--------------------------- photons -------------------------------------------*/
+
+value_identifier( float_X, MassRatioPhotons, 0.0 );
+value_identifier( float_X, ChargeRatioPhotons, 0.0 );
+
+using ParticleFlagsPhotons = bmpl::vector<
+    particlePusher< particles::pusher::Photon >,
+    shape< UsedParticleShape >,
+    interpolation< UsedField2Particle >,
+    massRatio< MassRatioPhotons >,
+    chargeRatio< ChargeRatioPhotons >
+>;
+
+/* define species photons */
+using PIC_Photons = Particles<
+    bmpl::string< 'p', 'h' >,
+    DefaultParticleAttributes,
+    ParticleFlagsPhotons
+>;
+
+
+/*--------------------------- ions -------------------------------------------*/
+
+/* ratio relative to BASE_CHARGE and BASE_MASS */
+value_identifier(float_X, MassRatioIons, 359100);
+value_identifier(float_X, ChargeRatioIons, -79.0);
+value_identifier(float_X, DensityRatioIons, 1.0);
+
+/*! Specify (chemical) element
+ *
+ * Proton and neutron numbers define the chemical element that the ion species
+ * is based on. This value can be non-integer for physical models taking
+ * charge shielding effects into account.
+ * @see http://en.wikipedia.org/wiki/Effective_nuclear_charge
```

Comment:
please remove this block, use https://github.com/ComputationalRadiationPhysics/picongpu/pull/1855 I will quickly add gold for you there